tasks/check-data-keys/check-data-keys.yaml

```diff
@@ -4,17 +4,23 @@ kind: Task
 metadata:
   name: check-data-keys
   labels:
-    app.kubernetes.io/version: "0.9.2"
+    app.kubernetes.io/version: "1.0.0"
   annotations:
     tekton.dev/pipelines.minVersion: "0.12.1"
     tekton.dev/tags: release
 spec:
   description: >-
-    Tekton task to check that all required information is present to use the specified system(s)
+    Tekton task to check that all required information is present to use the specified system(s) 
+    and the data is valid against the schema.
   params:
     - name: dataPath
       description: Path to the JSON string of the merged data to use
       type: string
+    - name: schema 
+      description: URL to the JSON schema file to validate the data against
+      type: string
+      # TODO: Update the default URL to the correct one 
```

Comment:
You could also update the schema in a separate PR that we merge first

Reply:
I think that would make more sense. WIll do it Monday

Reply:
https://github.com/konflux-ci/release-service-catalog/pull/630